.github/workflows/tests-pytest.yml

```diff
@@ -0,0 +1,34 @@
+name: Pytest
+
+on: push
+
+jobs:
+  run:
```

Comment:
Sorry, one change - can we make this `pytest` or something - easier to reference in required checks and other places.

Reply:
Oh, strange it doesn't include the workflow name. Done!